src/cli.rs

```diff
@@ -10,7 +10,11 @@ use tlparse::{parse_path, ParseConfig};
 #[command(author, version, about, long_about = None)]
 #[command(propagate_version = true)]
 pub struct Cli {
-    path: PathBuf,
+    #[arg(required_unless_present = "latest")]
+    path: Option<PathBuf>,
```

Comment:
Minor request: instead of making path optional, could you just keep it required (so no default to current directory; this is rarely what you want anyway since TORCH_TRACE makes you specify an output directory for logs)